packages/nextjs/package.json

```diff
@@ -5,7 +5,7 @@
   "types": "dist/index",
   "license": "MIT",
   "dependencies": {
-    "@appsignal/types": "^2.1.5",
+    "@appsignal/nodejs": "^2.0.0",
```

Comment:
This will need to be updated now that 2.1.0 has been released. In **all** `package.json` files.